src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -907,33 +911,48 @@ def _validate_and_store_performance_problem(self):
         exceeds_count_threshold = len(self.consecutive_db_spans) >= self.settings.get(
             "consecutive_count_threshold"
         )
-        if exceeds_count_threshold and self._are_db_spans_independant(self.consecutive_db_spans):
+        independant_db_spans = self._find_independant_spans(self.consecutive_db_spans)
```

Comment:
spelling nit: `independent` (variable and method name)

Reply:
Fixed!

Reply:
I actually make this mistake a lot lol